panel/src/components/Dialogs/Elements/Fields.vue

```diff
@@ -45,6 +45,12 @@ export default {
 	mixins: [props],
 	emits: ["input", "submit"],
 	computed: {
+		formValue() {
+			return {
+				...this.$helper.field.form(this.fields),
```

Comment:
`Question:` Why is this needed?

Reply:
Good question.

For this line:
https://github.com/getkirby/kirby/blob/lab/afbora/support-array-language-variables/config/areas/languages/dialogs.php#L55

Otherwise default value of `multiple` field is `null`. We have two condition fields `value` and `entries`. When `multiple` disabled `value` field visible and when enabled `entries` field visible. 

Not sure this is a correct solution.

Reply:
What if we don't use `default` there but rather add here https://github.com/getkirby/kirby/blob/lab/afbora/support-array-language-variables/config/areas/languages/dialogs.php#L202-L205 something like

```php
'value' => [
  'multiple' => false
]
```

Reply:
Yes, it seems to be working. 

What I'm wondering is; does this change I made really solve the problem of default values ​​in dialogs? Maybe we can use it in other dialogs or in the future.

Reply:
@distantnative What do you think?

Reply:
To be honest, I am not sure if your change is what is expected for dialogs - or rather the previous behavior. To get the entries field enhancement in v5.0 I would vote for reverting the `this.$helper.field.form()` part. We can do this in another PR later as a general improvement. But to get this PR merged, it would help if we don't reach into other areas that broadly.